packages/theme-chalk/src/date-picker/picker-panel.scss

```
@@ -4,7 +4,7 @@
 @include b(picker-panel) {
   color: getCssVar('text-color', 'regular');
   background: getCssVar('bg-color', 'overlay');
-  border-radius: getCssVar('border-radius-base');
+  border-radius: getCssVar('popper', 'border-radius');
```

```suggestion
  border-radius: getCssVarWithDefault('popper-border-radius', getCssVar('border-radius-base'));
```
Backward compatibility, if we separate the panel, the popper does not exist.